.github/workflows/ci.yml

```diff
@@ -1,22 +1,36 @@
-name: 'CI'
+name: "CI"
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request: ~
 
 jobs:
   run-tests:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rubyversion: ['2.4', '2.5', '2.6', '2.7', '3.0']
+        rubyversion: ["2.5", "2.6", "2.7", "3.0", "3.1"]
     steps:
-      - uses: actions/checkout@v2
-      - name: set up ruby
+      - name: Checkout Repository
```

Comment:
"checkout" is a noun, "check out" is the verb form.

Reply:
Well...not _exclusively_.

> This action checks-out ...
> When checking out the repository ...

They also say "Clone this repo" in their comment on their test workflow. https://github.com/actions/checkout/blob/25a956c84d5dd820d28caab9f86b8d183aeeff3d/.github/workflows/test.yml#L33

Anyway, of course I'm torn on this because, on the one hand I hate that software developers don't know that checkout isn't a verb, and on the other hand it references git checkout, but on the other hand git should be ashamed of its horrendous command interface...

Reply:
😂  love it